platform/pocketbook/koreader.app

```diff
@@ -133,8 +133,8 @@ while [ "${RETURN_VALUE}" -ne 0 ]; do
         # With a little notice at the top of the screen, on a big gray screen of death ;).
         "${KOREADER_DIR}/fbink" -q -b -c -B GRAY9 -m -y 1 "Don't Panic! (Crash n°${CRASH_COUNT} -> ${RETURN_VALUE})"
         if [ ${CRASH_COUNT} -eq 1 ]; then
-            # Warn that we're waiting on a tap to continue...
-            "${KOREADER_DIR}/fbink" -q -b -O -m -y 2 "Tap the screen to continue."
+            # Warn that we're sleeping for a bit...
+            "${KOREADER_DIR}/fbink" -q -b -O -m -y 2 "KOReader will restart in 15 sec."
```

Comment:
Strikes me as a bit odd compared to `15 s` or `15 seconds`.

Reply:
I, err, couldn't remember what we'd settled on in KoboUSBMS :D.

Reply:
But as it turns out, it happens to be 'sec', too ;).